src/apps/MultitouchExtension/src/FingerManager.swift

```diff
@@ -118,35 +129,51 @@ class FingerManager: ObservableObject {
         continue
       }
 
-      if !s.touchedFixed {
-        continue
-      }
+      if s.touchedFixed {
+        if s.touchedPhysically {
```

Comment:
We need to increment the count even if touchedPhysically is false while touchedFixed is true.
This happens when the release detection delay is set.

<img width="900" alt="Screenshot 2023-07-29 at 16 02 49" src="https://github.com/pqrs-org/Karabiner-Elements/assets/659178/96bc2119-9fe7-4304-b443-1474f6b7efcc">